### DISCUSSION TITLE
Show number of reaction in the bubble

### DISCUSSION THREAD

**Discussion by Abacaxi-Nelson** [GetStream/stream-chat-flutter#804] [General]
Hello
Can you point me where to find the number of reaction on a message.
For instance, if two users like (heart) a message, i want to get the count of likes (2) to show next the svg.

```dart
ReactionBubble paint the bubble
im stuck to finding the count
```

Thank you,
Nelson :)

**Comment by imtoori** [comment]
Hi Nelson and sorry for the delay, but discussions don't have slack notifications yet, so it's hard for us to get notified
Did you manage to do it?
The number of reactions is available in `message.reactionCounts`
The total number of reaction (considering that a user can send more than 1 reaction of the same type, but this depends on your implementation) is available in `message.reactionScores`
